sktime/forecasting/base/_base.py

```diff
@@ -1408,6 +1411,7 @@ def _check_fh(self, fh):
         Reads and writes to self._fh
         Writes fh to self._fh if does not exist
         Checks equality of fh with self._fh if exists, raises error if not equal
+        Assigns the frequency inferred from self._y
```

Comment:
to where/what?

Reply:
added "to the returned forecasting horizon object"

Reply:
but then this is in the wrong section, this is the section about what it does to `self`

Reply:
sorry, I am confused. your question was "to where/what?". I have added the end of the sentence to answer this question. What am I missing?